common/_global.scss

```diff
@@ -0,0 +1,12 @@
+*, *:before, *:after {
+  -moz-box-sizing: border-box;
```

Comment:
Each selector in a comma sequence should be on its own line<br>Avoid vendor prefixes.